zone/command.cpp

```diff
@@ -2471,18 +2471,29 @@ void command_grid(Client *c, const Seperator *sep)
 		/**
 		 * Spawn grid nodes
 		 */
+		std::map<std::vector<float>, int32> zoffset;
+
 		for (auto row = results.begin(); row != results.end(); ++row) {
-			auto node_position = glm::vec4(atof(row[0]), atof(row[1]), atof(row[2]), atof(row[3]));
-
-			NPC *npc = NPC::SpawnGridNodeNPC(
-				target->GetCleanName(),
-				node_position,
-				static_cast<uint32>(target->CastToNPC()->GetGrid()),
-				static_cast<uint32>(atoi(row[4])),
-				static_cast<uint32>(atoi(row[5]))
-			);
-			npc->SetFlyMode(GravityBehavior::Flying);
-			npc->GMMove(node_position.x, node_position.y, node_position.z, node_position.w);
+			glm::vec4 node_position = glm::vec4(atof(row[0]), atof(row[1]), atof(row[2]), atof(row[3]));
+
+			std::vector<float> node_loc {
+					node_position.x, node_position.y, node_position.z
```

Comment:
Indent seems to be off here; can we newline each of the values?

Example

```
std::vector<float> node_loc {
	node_position.x, 
	node_position.y, 
	node_position.z
};
```

Reply:
Done.